src/neuroconv/datainterfaces/behavior/lightningpose/lightningposeconverter.py

```diff
@@ -106,7 +115,6 @@ def add_to_nwbfile(
         metadata: dict,
         reference_frame: str | None = None,
         confidence_definition: str | None = None,
-        external_mode: bool = True,
```

Comment:
I'd like to deprecate this parameter for six months first before removing it outright.